### ISSUE TITLE
SubfieldBase deprecated in Django 1.10

### ISSUE THREAD

**Issue by robmoggach** [redouane/django-fontawesome#17]
Here's the exception:

```
fontawesome/fields.py:7: RemovedInDjango110Warning: SubfieldBase
has been deprecated. Use Field.from_db_value instead.
```

**Comment by frague59** [comment]
I'm now using django 1.10, but I can't use the application anymore... due to the use of SubfieldBase if IconField declaration.

**Comment by redouane** [comment]
hi, this will be fixed soon, you can send a pr if you want this now

**Comment by sidneijp** [comment]
I sent a pull request with the fix:
https://github.com/redouane/django-fontawesome/pull/26

I didn't tested with previous versions of Django, it's just a fix with Django 1.10+ in mind.

**Comment by redouane** [comment]
merged, thanks.
